plugins/autowarmth.koplugin/main.lua

```diff
@@ -481,19 +481,18 @@ function AutoWarmth:getLocationMenu()
                 info_text = _("Enter decimal degrees, northern hemisphere and eastern length are '+'."),
                 left_text = _("Latitude"),
                 left_value = self.latitude,
-                left_default = 0,
```

Comment:
Why was that one removed ? It seems the default value is "nil" ?

Dunno if that comment applies here, but in general, even if the default was 0, I would prefer seeing it duplicated here - to not have to go look into DoubleSpinWidget for what is its default.

Reply:
As I understand, it is a leftover. There should not be default values and there is no default button (default geographical location?).
In DSW the default button is switched with boolean (which was absent in the above dialog).
https://github.com/koreader/koreader/blob/90abcb022f6ae24d08e4c1f725024604faa76e50/frontend/ui/widget/doublespinwidget.lua#L53

Reply:
If default values are needed there, the callback must handle the nil `lat` and `long`
https://github.com/koreader/koreader/blob/c7a3c17ce06668137e6bcb231315937a401e5abc/plugins/autowarmth.koplugin/main.lua#L497-L499
like in
https://github.com/koreader/koreader/blob/90abcb022f6ae24d08e4c1f725024604faa76e50/frontend/apps/reader/modules/readerdevicestatus.lua#L193-L197

Reply:
> As I understand, it is a leftover. 

Oh, alright then :)